core/cloudflow-akka/src/main/scala/cloudflow/akkastream/AkkaStreamletContextImpl.scala

```diff
@@ -67,7 +67,8 @@ final class AkkaStreamletContextImpl(
 
   private val bootstrapServers = system.settings.config.getString("cloudflow.kafka.bootstrap-servers")
 
-  def sourceWithOffsetContext[T](inlet: CodecInlet[T]): cloudflow.akkastream.scaladsl.SourceWithOffsetContext[T] = {
+  // internal implementation that uses the CommittableOffset implementation to provide access to the underlying offsets
+  private[akkastream] def _sourceWithContext[T](inlet: CodecInlet[T]): SourceWithContext[T, CommittableOffset, _] = {
```

Comment:
no need for '_' ?

Reply:
I like to signal that this is for internal use only.

Reply:
the `private` gives it away right ;-)

Reply:
We don't use this naming convention so far so rather not do it.

Reply:
I see we started to use it in a few places. I'm not too happy with the style but if you _really_ like it a lot it is fine.

Reply:
I prefer the "_" for internal impl for internal implementations of like-named public methods. But I can also go without it. The thinking behind is that you could have:
```
 def f(...) ... // public
 def _f // private implementation of f
```
you will also see me using it for recursive implementations, like:
``` 
def doThis() {
  def _doThis {//really do this
  
 }
  _doThis(init value)
}
```

Reply:
but I removed the `_` ...  let's move on and merge this.

Reply:
yeah I should have not said anything TBH @maasg I need to get rid of these unimportant preferences...

Reply:
So in the future feel free to do this.